### DISCUSSION TITLE
What happened to issue #605?

### DISCUSSION THREAD

**Discussion by HighCommander4** [clangd/clangd#1177] [General]
There was a clangd issue at https://github.com/clangd/clangd/issues/605, titled "clangd should respect provided rootUri", which contained a fair amount of discussion related to the topic of clangd making use of the `rootUri` provided in the `initialize` request.

I have notifications related to this issue in my email archives.

I would like to review this issue, and point users with relevant questions to that discussion.

What happened to this issue? The link gives me a "Page not found".

**Comment by sam-mccall** [comment]
This is mysterious to me, and I don't have an explanation.

Issues can be deleted, resulting in this 404 page ([example](https://github.com/sam-mccall/dotfiles/issues/1)), but:
 - only repository admins can delete issues
 - the only repository admins are @kadircet and myself (via the @clangd org)
 - the org-level "delete issues" setting is off and defaults to off (an admin would have had to turn it on, delete issue, turn it off)

I've requested logs for the repo (the last notification I have from that issue is November 2020, so from then until now).
I've also opened a ticket with github support to see if they can say what happened (https://support.github.com/ticket/personal/0/1650009 - just for my reference, I expect nobody else can access it). This isn't a paid account/org of any kind so I'm not sure what kind of support to expect.

Thanks for flagging this, I'll let you know what I find.

**Comment by sam-mccall** [comment]
I got an initial response: it relates to the initial opener of the issue, and they can't say more for privacy reasons. (This response came quickly, i just missed the notification)

So for example (complete speculation on my part) this might mean that the reporter requested their personal data be deleted under GDPR, and so issues they filled were removed completely.

As most of the content was from other users (i assume) and this is valuable, this seems like a poor outcome and frankly something of a violation of trust in GitHub as stewards of our data.
I've reopened the ticket to ask if any of the data can be restored/provided, and for docs around these policies. Will follow up.
